setup.md

```diff
@@ -284,6 +257,9 @@ To download the data, run the following command using [wget](https://en.wikipedi
 wget --no-clobber --input-file \
   https://github.com/ESMValGroup/ESMValTool_Tutorial/raw/master/data/dataset.urls \
   --directory-prefix $HOME/esmvaltool_tutorial/data/
+cd $HOME/esmvaltool_tutorial/data/
```

Comment:
Could you explain why these (mkdir/mv) lines are needed in the setup.md? Maybe in separate text + code block

Reply:
Thanks. I added some explanations.

Reply:
great